react/features/video-layout/middleware.native.js

```diff
@@ -18,6 +20,13 @@ MiddlewareRegistry.register(store => next => action => {
     case SET_CAR_MODE:
         dispatch(setVideoMuted(action.enabled, MEDIA_TYPE.VIDEO, VIDEO_MUTISM_AUTHORITY.CAR_MODE));
         break;
+    case CLIENT_RESIZED: {
+        const { clientHeight, clientWidth } = store.getState()['features/base/responsive-ui'];
+
+        // On mobile the large video should always fill the screen.
+        dispatch(setLargeVideoDimensions(clientHeight, clientWidth));
```

Comment:
Yes this is used only for the receiver constraints. When I was discussing this with @saghul we agreed to take a shortcut here because the height of the large video will match the height of the screen in 99% of the cases. I think the only use case where it won't match is if we are displaying screen sharing and since we have pinch to zoom there better quality will be beneficial.

Reply:
I think the reverse is going to happen from what I understand about viewport sizes and screen resolutions. For example, the viewport size for iPad Pro is 1024 x 1366 while the actual resolution is about 2048 x 2732 Pixels since it has a pixel density of 2. So we will end up requesting lower resolutions. I don't think it matters much since we set the preferred video quality to 360p on mobile IIRC but again @saghul should be able to confirm.

Reply:
Now I see what you meant. 
So do you think in your example we need to request 2*height of video? 

Do we currently respect this anywhere? I don't think mobile web respects this currently as well.

You are correct that for react-native it doesn't matter in practice because preferred video quality is set to 360. But since this is an external parameter it is better to handle such things here.

Anyway if we don't handle this use case anywhere lets treat it as a new feature in a new PR? WDYT?